sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample6_WrapUnwrap.md

```diff
@@ -0,0 +1,73 @@
+# Wrapping and unwrap a key
+
+This sample demonstrates how to wrap and unwrap a symmetric key with an RSA key.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](../README.md) for links and instructions.
+
+## Creating a KeyClient
```

Comment:
Seems like this whole section "Creating a KeyClient" can be put in an include

Reply:
Doesn't work here.